Makefile

```diff
@@ -1,14 +1,17 @@
-.PHONY: all build test
+.PHONY: all build unit-tests integration-tests
```

Comment:
There's not a lot of tests, can we keep just build + tests?

Reply:
The main reason for separating those was that users expect the project to build fast. As part of the integration tests we spin up a target server which is time-consuming.

Also, planning to add more end-to-end tests and validation around those so better to do this now.